UOP1_Project/ProjectSettings/QualitySettings.asset

```diff
@@ -21,7 +21,7 @@ QualitySettings:
     skinWeights: 2
     textureQuality: 0
     anisotropicTextures: 1
-    antiAliasing: 0
+    antiAliasing: 2
```

Comment:
Please avoid changing quality settings when they are not related to the PR

Reply:
Reverted to original state